flytekit/common/tasks/sagemaker_task.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import absolute_import
+
+import datetime as _datetime
+from flytekit.sdk import types as _sdk_types
+from flytekit.common.tasks import task as _sdk_task
+from flytekit.common import interface as _interface
+from flytekit.models import (
+    literals as _literal_models,
+    task as _task_models,
+    interface as _interface_model,
+)
+from google.protobuf.json_format import MessageToDict, ParseDict
+import gen.pb_python.sagemaker_pb2 as sagemaker_pb2
+from flytekit.common.exceptions import scopes as _exception_scopes
+
+_TASK_TYPE = "aws_sagemaker_hpo"
```

Comment:
you want to use the declared constant?

Reply:
Could you explain what you mean? I don't get the question